### PR TITLE
Create logs-dashboard.md

### PR DESCRIPTION
**Pull request by xuelangos**

### CODE REVIEW

**Review by wenchajun — APPROVED** [review]

**Comment by faweizhao26** [comment]
@zhuxiujuan28

File: content/zh/docs/v3.4/cluster-administration/logs-dashboard.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+1. 使用 `admin` 用户登录控制台，点击左上角的**平台管理**，选择**集群管理**。
+
+2. 点击**定制资源定义**，在搜索栏中输入 `clusterconfiguration`，点击结果查看其详细页面。
+
+    {{< notice info >}}
+定制资源定义（CRD）允许用户在不增加额外 API 服务器的情况下创建一种新的资源类型，用户可以像使用其他 Kubernetes 原生对象一样使用这些定制资源。
+    {{</ notice >}}
+
+3. 在**自定义资源**中，点击 `ks-installer` 右侧的![](https://hackmd.io/_uploads/r1cUCTap3.png)
```
Comment (zhuxiujuan28) [inline content/zh/docs/v3.4/cluster-administration/logs-dashboard.md:120]: 所有图片放到本地路径下：website/static/images/docs/v3.4

**Pull request merged**